config/express.js

```diff
@@ -51,7 +51,7 @@ passport.serializeUser(function(user, done) {
 });
 
 passport.deserializeUser(function(name, done) {
-  User.findOne({name}, function(err, user) {
+  User.findOne({id: name}, function(err, user) {
```

Comment:
So, this isn't even valid JS :-P

I'm assuming you meant User.findOne(name

But anyway, I've made it explicit.